tests/test_orient_symmetric.py

```diff
@@ -46,7 +52,10 @@ def setUp(self):
             )
 
             self.cl_orient_ests[order] = CLSymmetryC3C4(
-                self.srcs[order], symmetry=f"C{order}", n_theta=self.n_theta
+                self.srcs[order],
+                symmetry=f"C{order}",
+                n_theta=self.n_theta,
+                max_shift=1 / self.L,
```

Comment:
Getting head start on this.

Did you want this to run for shifts?
`shift_step=1`
` self.max_shift = math.ceil(max_shift * self.n_res)` # 1
...
`n_shifts = int(np.ceil(2 * max_shift / shift_step + 1))` # 2*1/1 +1

Reply:
As discussed in the meeting, we're going to leave this to exercise searching over shifts. It only adds a couple seconds over testing without shifts.